integrated_channels/integrated_channel/management/commands/__init__.py

```diff
@@ -0,0 +1,101 @@
+"""
+Enterprise Integrated Channel management commands.
+"""
+from __future__ import absolute_import, unicode_literals
+
+from django.core.management.base import CommandError
+from django.utils.translation import ugettext as _
+
+from enterprise.models import EnterpriseCustomer
+from integrated_channels.sap_success_factors.models import SAPSuccessFactorsEnterpriseCustomerConfiguration
+
+
+# Import djcelery, or stub it if not available.
+try:
+    from djcelery.celery import task as celery_task
+except ImportError:
+    def celery_task(func):
```

Comment:
@e-kolpakov I would much rather use the real libraries, and not have to mock all this stuff!

@brittneyexline @mattdrayer @haikuginger Wanna weigh in?